core_backend/app/llm_call/parse_input.py

```diff
@@ -182,22 +230,13 @@ async def _translate_question(
             )
         )
 
-    if question.original_language == IdentifiedLanguage.UNKNOWN:
-        supported_languages = ", ".join(IdentifiedLanguage.get_supported_languages())
-
-        error_response = UserQueryResponseError(
-            error_message=STANDARD_FAILURE_MESSAGE
-            + f" Only the following languages are supported: {supported_languages}. ",
-            query_id=response.query_id,
-            error_type=ErrorType.UNKNOWN_LANGUAGE,
-        )
-
-        logger.info(
-            "TRANSLATION FAILED due to UNKNOWN language on query id: "
-            + str(response.query_id)
-        )
+    # In case translation is run on an invalid language question
```

Comment:
We can rethink this later but thought it wouldn't hurt to add the check here..

Reply:
Don't we do this check in the identify_language() function anyway? Why twice?

Reply:
Oh i guess we already check if identify language has been called above this, so yeah this is redundant!